utils/template_renderer.js

```diff
@@ -0,0 +1,8 @@
+import Mustache from 'mustache'
+
```

Comment:
Just to understand this does this essentially inject data variables into json and return it as a javascript object?

Reply:
Yes, this file does that.  Mustache just takes a template as a string, and it injects values where the `{{ thing }}` tokens are using an object for the token-value definitions.